### ISSUE TITLE
brew failed: Download failed: $STABLE_URL

### ISSUE THREAD

**Issue by drnic** [buildkite/cli#83]
```
$ brew upgrade
...
==> Upgrading buildkite/cli/bk v1.0.0 -> v1.1.0
==> Downloading $STABLE_URL
Warning: Transient problem: timeout Will retry in 1 seconds. 3 retries left.
Warning: Transient problem: timeout Will retry in 2 seconds. 2 retries left.
Warning: Transient problem: timeout Will retry in 4 seconds. 1 retries left.
##O=#  #
curl: (6) Could not resolve host: $STABLE_URL
Error: Failed to download resource "bk"
Download failed: $STABLE_URL
```

Looks like https://github.com/buildkite/cli/blob/master/.github/tap-release.yml#L11 `$STABLE_URL` and `$STABLE_SHA256` are not being replaced with a value; but `$STABLE_VERSION` is correctly being given a value.

My local `bk` formula is:

```
# Generated by https://github.com/buildkite/cli/blob/master/.github/tap-release.yml

class Bk < Formula
  homepage "https://github.com/buildkite/cli"
  desc "A command line interface for Buildkite."
  url "$STABLE_URL"
  version "v1.1.0"
  sha256 "$STABLE_SHA256"

  def install
    mv "bk-darwin-amd64-1.1.0", "bk"
    bin.install "bk"
  end

  test do
    assert_match "bk #{version}", shell_output("#{bin}/bk --version 2>&1")
  end
end
```

**Comment by samattyro** [comment]
Is there any update for fixing or workaround please.

**Comment by orien** [comment]
The examples in the [`tap-release`](https://github.com/toolmantim/tap-release) readme specify slightly different variable names.

```diff
-    url      "$STABLE_URL"
-    sha256   "$STABLE_SHA256"
+    url      "$STABLE_ASSET_URL"
+    sha256   "$STABLE_ASSET_SHA256"
```

Could this be the fix?

**Comment by samattyro** [comment]
Thank orien.  I manually updated my bk.rb file to "$STABLE_ASSET_URL" and "$STABLE_ASSET_SHA256" with no luck.  I feel they should be replaced with values as the variable 'version' does.

**Comment by orien** [comment]
@samattyro You should be able to manually update your `bk.rb` file with the following values as a work around:

```ruby
url "https://github.com/buildkite/cli/releases/download/v1.1.0/bk-darwin-amd64-1.1.0"
sha256 "580e595c67be43d95ecfd4525c0d0070ea85e8d288370af56ae6d3163c5164f3"
```

**Comment by samattyro** [comment]
@orien Man, you are the champion.  Thanks a lot.

**Comment by pda** [comment]
I think this was fixed in #85?
